src/lib/SyntaxData.ml

```diff
@@ -45,5 +48,6 @@ and tp =
   | Sg of tp * tp
   | Id of tp * t * t
   | Nat
+  | Path of tp * t * t (*todo/iev: not quite sure i really understand why this should be in tp not t, given the relative flatness of coresyntaxdata.t*)
```

Comment:
This line is unnecessary; the existing path type code is just going to be equipped with an equation like `El(path(A,a)) == (i : dim) -> [blah blah blah]`; this means, for instance, that there is also no need for `PathAp` or `PathLam`, nor is there a need for concrete syntax forms for path lambda/path application, since they will come already from the function type.

Reply:
per discussion the other day: got it